grails-app/services/com/recomdata/transmart/data/export/HighDimExportService.groovy

```diff
@@ -6,68 +6,96 @@ import org.transmartproject.core.dataquery.highdim.AssayColumn
 import org.transmartproject.core.dataquery.highdim.HighDimensionDataTypeResource
 import org.transmartproject.core.dataquery.highdim.assayconstraints.AssayConstraint
 import org.transmartproject.core.dataquery.highdim.projections.Projection
-import org.transmartproject.db.dataquery.highdim.assayconstraints.PlatformConstraint
 import org.transmartproject.export.HighDimExporter
 
 class HighDimExportService {
 
     def highDimensionResourceService
     def highDimExporterRegistry
+    def queriesResourceService
 
     // FIXME: jobResultsService lives in Rmodules, so this is probably not a dependency we should have here
     def jobResultsService
 
     def exportHighDimData(Map args) {
-        String jobName = args.jobName
-        String dataType = args.dataType
-        def resultInstanceId = args.resultInstanceId
+        Long resultInstanceId = args.resultInstanceId as Long
         List<String> conceptPaths = args.conceptPaths
-        String studyDir = args.studyDir
-        Collection<String> gplIds = args.gplIds
-        String format = args.format
 
-
-        if (jobIsCancelled(jobName)) {
+        if (jobIsCancelled(args.jobName)) {
```

Comment:
I think so too. But it's here because it's used inside actual export (most time consuming part) as well. See inside this module the line line `exporter.export tabularResult, projection, outputStream, { jobIsCancelled(jobName) }`